politeiawww/cmd/piwww/piwww.go

```diff
@@ -80,8 +79,6 @@ type piwww struct {
 	Me                 shared.MeCmd             `command:"me" description:"(user)   get user details for the logged in user"`
 	NewUser            NewUserCmd               `command:"newuser" description:"(public) create a new user"`
 	Policy             PolicyCmd                `command:"policy" description:"(public) get the server policy"`
-	ProposalDetails    ProposalDetailsCmd       `command:"proposaldetails" description:"(public) get the details of a proposal"`
-	ProposalPaywall    ProposalPaywallCmd       `command:"proposalpaywall" description:"(user)   get proposal paywall details for the logged in user"`
```

Comment:
Proposal paywall isn't actually a proposals route. Its misnamed. It's a user route since the paywall is stored as user data. Don't detail the paywall routes.

Reply:
added proposal paywall back mate. will factor it out to be a user route on a following PR.